src/main/java/org/mdbenefits/app/inputs/MdBenefitsFlow.java

```diff
@@ -176,11 +182,19 @@ public class MdBenefitsFlow extends FlowInputs {
 
     private String migrantOrSeasonalFarmWorkerInd;
 
-    private String citizenshipInd;
+    @NotBlank
+    private String allAreCitizens;
 
-    private String nonCitizens;
+    @NotBlank
+    private String applicantCitizenshipStatus;
 
-    private String citizenshipNumber;
+    @DynamicField
+    private String citizenshipStatus;
+
+    // The previous field (citizenshipStatus) is used as a wildcard to help populate
+    // this field below, which is put in the "household[]" members data and not left as
+    // a top-level field in the json.
+    private String householdMemberCitizenshipStatus;
```

Comment:
These variables will change some if they revamp the pages.  I wouldn't worry about these too much.